templates/argocd-webhook-ingress.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: argocd-webhook-ingress-push
```

Comment:
argocd-server might be better so when we get the appset working it's clear what's for what